config/zones/US.yaml

```diff
@@ -96,6 +96,7 @@ subZoneNames:
   - US-SW-WALC
   - US-TEN-TVA
   - US-TEX-ERCO
+  - US-SE-SOCO
```

Comment:
It might be better to keep the ordering, so have it right after US-SE-SOCO.

Reply:
Ah yeah, I'll move it.